src/myr/Myr.js

```diff
@@ -153,6 +153,53 @@ class Myr {
         }
     };
 
+    increasePosition = (x = 0, y = 0, z = 0) => {
+
+        if (typeof x === "number" && typeof y === "number" && typeof z === "number") {
```

Comment:
In case the user also wants to use the new position for another purpose, let's also return the new position for each of these.

Reply:
(So the position for increasePosition and the x, y, or z coordinate for the rest of them.

Reply:
Should the return type for the increasePosition() be an array or object? I think array is better since you can use the array as an argument using the spread operator but what you think? 

Ex. 
`let x = increasePosition(2,2,2);`
`setPosition(...x); //same position`

Reply:
Let's return it as an object. If they're setting the position twice, that's on them. There isn't really a reason to do that. I was thinking more for animations where as you go up in the y axis you might increase your width. There is a scene that does this but I can't think of it. A user might want to use the new y coordinate to specify the width of the next entity.